video/out/gpu/lcms.h

```diff
@@ -42,6 +42,11 @@ bool gl_lcms_has_changed(struct gl_lcms *p, enum mp_csp_prim prim,
 
 static inline bool gl_parse_3dlut_size(const char *arg, int *p1, int *p2, int *p3)
 {
+    if (!arg) {
+        *p1 = *p2 = *p3 = 0;
+        return true;
+    }
```

Comment:
shouldn't be necessary anymore should it?

Reply:
Oops, you're right, that was left over